 package com.examples.typeuseannotations;

 /** testing. */
-public class Checkstyle {
+public class GoogleFormatter {
   final @TypeUseAnnotation String name = "xyz";

   @TypeUseAnnotation final int xyz = 90;

-  @TypeUseAnnotation public int test() {
+  @TypeUseAnnotation
+  public int test() {
     return 0;
   }

@@ -14,10 +15,3 @@ public class Checkstyle {
     return 10;
   }
 }
-
-/*
- * CLI OUPUT:-
- * java -jar ..\checkstyle-10.17.0-all.jar -c ..\google_checks.xml .\Checkstyle.java
- * Starting audit...
- * Audit done.
- */
